design-documents/graph-ql/coverage/company.md

```diff
@@ -36,7 +36,10 @@ type Company @doc(description: "Company entity output data schema.") {
     ): CompanyRoles!  @doc(description: "Returns the list of defined roles at Company.")
     role(id: ID!): CompanyRole  @doc(description: "Returns company role by id.")
     acl_resources: [CompanyAclResource]  @doc(description: "Returns the list of all permission resources.")
-    hierarchy: CompanyHierarchyOutput  @doc(description: "Returns the complete data about company structure.")
+    structure(
```

Comment:
Not something to fix for company. Do we want to have pagination in tree queries in other places like categories?

Reply:
Ideally, but think it's still open for discussion internally. See wiki doc titled `Representing Trees in GraphQL`